sh/tests/right_test.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+
+FILENAME="student/right.sh"
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
```

Comment:
Here we could add the necessary boilerplate for the script to make tests, like:
```sh
mkdir right
touch right/a_file.txt right/txt_file right/txt right/example right/example.doc right/example.TXT right/example.txtnot
```

like so we don't have example files for tests in the repo, and we see the logic more easily in the testing script.

Reply:
fixed